tests/unit/TechRecordsDao.unitTest.ts

```diff
@@ -143,6 +147,119 @@ describe("TechRecordsDAO", () => {
       });
     });
   });
+
+  context("createSingle", () => {
+    context("builds correct request", () => {
+      beforeEach(() => {jest.resetModules(); });
+      afterEach(() => {sandbox.restore(); });
+
+      it("for valid TechRecord", async () => {
+        const expectedCall = {
+          TableName: "cvs-local-technical-records",
+          Item: techRecord,
+          ConditionExpression: "vin <> :vin AND partialVin <> :partialVin",
+          ExpressionAttributeValues: {
+            ":vin": "XMGDE02FS0H012345",
+            ":partialVin": "012345"
+          }
+        };
+        const stub = mockDocumentClientWithReturn("put", []);
+        const techRecordsDao = new TechRecordsDao();
+        await techRecordsDao.createSingle(techRecord);
+        expect(getRequestItemsBodyFromStub(stub)).to.deep.equal(expectedCall);
+      });
+
+      it("for invalid TechRecord", async () => {
+        delete techRecord.partialVin;
+        delete techRecord.vin;
+        const expectedCall = {
+          TableName: "cvs-local-technical-records",
+          Item: techRecord,
+          ConditionExpression: "vin <> :vin AND partialVin <> :partialVin",
+          ExpressionAttributeValues: {
+            ":vin": undefined,
+            ":partialVin": undefined
+          }
+        };
+        const stub = mockDocumentClientWithReturn("put", []);
+        const techRecordsDao = new TechRecordsDao();
+        await techRecordsDao.createSingle(techRecord);
+        expect(getRequestItemsBodyFromStub(stub)).to.deep.equal(expectedCall);
+      });
+    });
+
+    context("when creating a new vehicle", () => {
+      it("should be successful and return {}", async () => {
+        techRecord.vin = Date.now().toString();
+        techRecord.partialVin = techRecord.vin.substr(techRecord.vin.length - 6);
+        techRecord.primaryVrm = Math.floor(100000 + Math.random() * 900000).toString();
+        techRecord.techRecord[0].bodyType.description = "new tech record";
+        const techRecordsDao = new TechRecordsDao();
+        return techRecordsDao.createSingle(techRecord)
```

Comment:
This is an integration test (basically, anything without AWS SDK mocks). Could it, and those like it, please be moved to a separate file?